prow/plugins/plugins.go

```diff
@@ -160,6 +168,15 @@ func RegisterGenericCommentHandler(name string, fn GenericCommentHandler, help H
 	genericCommentHandlers[name] = fn
 }
 
+// GenericCommentHandler defines the function contract for a github.GenericCommentEvent handler.
```

Comment:
nit: Fixup the names in the godoc

Also, I think there should be some place that explains how are "Extended" variants different from the normal ones.,

Reply:
updated the comment and function signature